File: docs/book/Part F - Fundamentals/F1 - Programming and Remote Sensing Basics/F1.0 Javascript and the Earth Engine API/F10b Checkpoint.py
import ee 
import geemap

Map = geemap.Map()

#  ~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
#  Chapter:      F1.0 Exploring images
#  Checkpoint:   F10b
#  Author:       Ujaval Gandhi
#  ~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~

a = 1
b = 2

result = ee.Number(a).add(b)
print(result)

yearList = ee.List.sequence(1980, 2020, 5)
print(yearList)

#  -----------------------------------------------------------------------
#  CHECKPOINT
#  -----------------------------------------------------------------------

Map

File: docs/book/Part F - Fundamentals/F1 - Programming and Remote Sensing Basics/F1.0 Javascript and the Earth Engine API/F10b Checkpoint.js
//  ~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
//  Chapter:      F1.0 Javascript and the Earth Engine API
//  Checkpoint:   F10b
//  Author:       Ujaval Gandhi 
//  ~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~

var a = 1;
var b = 2;

var result = ee.Number(a).add(b);
print(result);

var yearList = ee.List.sequence(1980, 2020, 5);
print(yearList);

//  -----------------------------------------------------------------------
//  CHECKPOINT 
//  -----------------------------------------------------------------------

